src/content/login/index.tsx

```diff
@@ -1,70 +1,242 @@
+                                    <img src="/static/images/logo/google.svg" />
+                                }
+                                fullWidth
+                            >
+                                Log in with Google
                             </Button>
-                        </Link>
-                        <Link to="/signup">
-                            <Button variant="outlined" type="button" style={{ width: "50%" }}>
-                                Don't have an account?
-                            </Button>
-                        </Link>
-                    </Box>
+                        </Grid>
+                        <Grid item xs={12}>
+                            <Divider sx={{ mt: 2, mb: 2 }}>
+                                <Typography
+                                    variant="subtitle1"
+                                    sx={{ fontStyle: "italic" }}
+                                >
+                                    Or enter your credentials below to continue.
+                                </Typography>
+                            </Divider>
+                        </Grid>
+                        <Grid item xs={12}>
+                            <LoginForm />
+                        </Grid>
+                        <Grid item xs={12}>
+                            <Divider sx={{ mt: 2, mb: 2 }} />
+                        </Grid>
+                        <Grid item xs={12}>
+                            <ButtonGroup fullWidth>
+                                <Button
+                                    component={Link}
+                                    to="/reset-password"
+                                    variant="outlined"
+                                    type="button"
+                                    fullWidth
+                                >
+                                    Forgot password?
+                                </Button>
+                                <Button
+                                    component={Link}
+                                    to="/signup"
+                                    variant="outlined"
+                                    type="button"
+                                    fullWidth
+                                >
+                                    Looking for signup?
+                                </Button>
+                            </ButtonGroup>
+                        </Grid>
+                    </Grid>
                 </Card>
-            </Container>
-        </OverviewWrapper>
+            </Grid>
+        </Grid>
+    );
+};
+
+/**
+ * Component for the login form.
+ */
+const LoginForm = () => {
+    const [passwordVisible, setPasswordVisible] = useState(false);
+
+    const [rememberMe, setRememberMe] = useState(false);
+
+    const [usernameIR, passwordIR] = [
+        useRef<HTMLInputElement>(null),
+        useRef<HTMLInputElement>(null),
+    ];
```

Comment:
It is not recommended to use refs, instead make the components controlled. See https://reactjs.org/docs/forms.html#controlled-components (Note the docs are outdated and use class components, but the same logic still applies)